test/integration/test/ip_camera_test.js

```diff
@@ -5,12 +5,14 @@ const Config = require('config-js');
 const isoDate = require('iso-date');
 
 var config = new Config('./test/integration/lib/config/foxbox.js');
-var Prepper = require('../lib/testPrepper.js');
+var Prepper = require('../lib/make_suite.js');
 
 Prepper.makeSuite('Control camera locally',function(){
 
   var cameraService;
 
+  before(Prepper.turnAllSims);
```

Comment:
I think I can't use arrow functions within the hook, as the doc says: "Passing arrow functions to Mocha is discouraged. Their lexical binding of the this value makes them unable to access the Mocha context, and statements like this.timeout(1000); will not work inside an arrow function."

I did read that it supports mutiple before(), but that wasn't from the formal documentation. foxboxLogin() is a chain, but it is sync, so I don't think we need to worry about asnycs here.